src/protocols/secure_channel/CASESession.h

```diff
@@ -26,16 +26,14 @@
 #pragma once
 
 #include <credentials/CHIPCert.h>
-#include <crypto/CHIPCryptoPAL.h>
-#if CHIP_CRYPTO_HSM
-#include <crypto/hsm/CHIPCryptoPALHsm.h>
-#endif
 #include <credentials/CertificateValidityPolicy.h>
 #include <credentials/FabricTable.h>
 #include <credentials/GroupDataProvider.h>
+#include <crypto/CHIPCryptoPAL.h>
 #include <lib/core/CHIPTLV.h>
 #include <lib/core/ScopedNodeId.h>
 #include <lib/support/Base64.h>
+#include <lib/support/CHIPMem.h>
```

Comment:
Why do we need CHIPMem in the header here?

Reply:
At first I had a UniquePtr until it was proven that I can't use that for the purpose. Forgot to remove the header. Done now.